static/docs/commands-reference/repro.md

```diff
@@ -15,25 +15,83 @@ name is omitted, `Dvcfile` will be used by default.
     positional arguments:
         target                DVC file to reproduce.
 
-    optional arguments:
-        -h, --help            show this help message and exit
-        -q, --quiet           Be quiet.
-        -v, --verbose         Be verbose.
-        -f, --force           Reproduce even if dependencies were not changed.
-        -s, --single-item     Reproduce only single data item without recursive
-                              dependencies check.
-        -c CWD, --cwd CWD     Directory within your project to reproduce from.
-        -m, --metrics         Show metrics after reproduction
-        --dry                 Only print the commands that would be executed without
-                              actually executing.
-        -i, --interactive     Ask for confirmation before reproducing each stage.
-        -p, --pipeline        Reproduce the whole pipeline that the specified stage
-                              file belongs to.
 ```
 
+
+## Description
+
+`dvc repro` provides an interface to rerun the commands in the 
+computational graph (a.k.a. pipeline) defined by the stage files 
+in the current workspace.  By default, this command searches, 
+starting from the `Dvcfile`, the pipeline stages to find any
+which have changed.  It then reruns the corresponding commands.  The
+pipeline is defined using the `dvc run` command.
+
+There are several ways to limit the stages to rerun:
+
+* Listing stage file(s) as `targets`
+* The `--single-item` option ensures that only a single stage is rerun.
+* The `--pipeline` option limits execution to the pipeline
+ containing the named stage, especially useful if the
+ workspace contains multiple pipelines.  
+* The `--cwd` option limits execution to the pipeline 
+ located in a subdirectory.
+
+There are two ways to ensure rerunning additional stages beyond
+a changed stage:
+
+* The `--force` option causes the entire pipeline to be rerun
+* The `--ignore-build-cache` causes all stages following a changed stage
+ to be rerun.
+
 `dvc repro` does not run `dvc fetch`, `dvc pull` or `dvc checkout` to get source
 data files, intermediate or final results.
 
+## Options
+
+* `-q`, `--quiet` do not write anything to standard output.  The command
```

Comment:
can we move options like `-v`, `-q`, -h` to the very bottom of the list?

Reply:
Have updated the file to match